### DISCUSSION TITLE
Some questions

### DISCUSSION THREAD

**Discussion by 1lkei** [jlesage/docker-freefilesync#19] [Q&A] [answered]
I'm trying to use the command `docker exec -itd freefilesync sh -c '/opt/FreeFileSync/Bin/FreeFileSync /config/123.ffs_batch'` to run the program in the background for convenient use with other scripts. However, it doesn't execute. Only when I run `docker exec -it freefilesync sh` first and then run `/opt/FreeFileSync/Bin/FreeFileSync /config/123.ffs_batch` does it work. Is there a way to solve this? I don't know much about this. My English is not very good, so I used software to translate this.

**Comment by jlesage** [comment, marked as answer]
Try to add the `-i` option to `sh`:
```
docker exec -itd freefilesync sh -i -c '/opt/FreeFileSync/Bin/FreeFileSync /config/123.ffs_batch'
```

**Reply by 1lkei** [reply]
Thank you for your response, it has successfully run.
